configs/full.yaml

```diff
@@ -99,10 +92,27 @@ npz_fixed_field_keys:
 # key_mapping:
 #   free_energy: total_energy
```

Comment:
@nw13slx Can we document this better: it's not clear here which one is NequIP-internal and which one is VASP? Also not clear if this is required always? How does ASE parse it?